server/routes/handlers/getUpdateCommentRequestHandler.js

```diff
@@ -0,0 +1,9 @@
+const getUpdateCaseCommentHandler = caseService => async (req, res) => {
+  const { params: { courtCode, hearingId, defendantId }, body: { caseId, comment, commentId } } = req
+
+  await caseService.updateCaseComment(caseId, commentId, comment, res.locals.user.name)
```

Comment:
Should there be a try catch / with an error state? Question not a requirement

Reply:
I have suggested something on that lines to handle any unexpected errors and show some graceful error message to the user on this thread but doing so is not inline with the recommendation. Please see this thread https://mojdt.slack.com/archives/C01FR4HKS3A/p1686821180541949

So the catch block has no clue what to do with the caught error.

Reply:
yeah think there is a generic error handler that will catch any unknowns, but yes I think specific error handling could also be a thing to do

Reply:
Ok - I'll speak with the team about adding some work in the future about this.!